ui/src/main/resources/org/pentaho/mantle/public/messages/schedulerMessages_en.properties

```diff
@@ -687,4 +687,6 @@ paramsDialogNoMatches=No matching parameters...
 bulkDeleteResponseError=There was an error trying to remove the selected schedule(s). Please try again.
 bulkDeleteErrors=Some schedule(s) couldn't be removed. Please review the following:<br/><br/>{0}
 bulkDeleteSuccess=Schedule(s) successfully removed.
+bulkDeletePermissionDeniedError=Permission denied
+bulkDeleteUnknownError=Unable to delete. Please contact administrator
```

Comment:
"(...). Please contact your Administrator." is used in other places, we can use it here also.